ml-proto/test/func_ptrs.wast

```diff
@@ -0,0 +1,27 @@
+(module
+    (types
+        (func (param) (result i32))
+        (func $T (param i32) (result i32))
+        (func $U (param i32))
+    )
+
+    (func $one (type 0) (i32.const 13))
+    (export "one" $one)
+
+    (func $two (type $T) (i32.add (get_local 0) (i32.const 1)))
+    (export "two" $two)
+
+    (func $three (type $T) (param $a i32) (result i32) (i32.sub (get_local 0) (i32.const 2)))
```

Comment:
Could you add a comment here confirming to the reader who may not have read all the rules elsewhere that it's explicitly ok to have both a type and params/results, provided they match, because this lets the params be named?